test/OpenTelemetry.Extensions.AzureMonitor.Tests/OpenTelemetry.Extensions.AzureMonitor.Tests.csproj

```diff
@@ -1,7 +1,8 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
whitespace change?

Reply:
@mic-max Do you have any tips for how to view what changed here?
I didn't knowingly make any whitespace changes

Reply:
I've looked at Notepad++, a hex editor, and WinMerge. I can't tell what changed here. :)

To get this resolved, i'll checkout the version from main and reapply my framework changes.

Reply:
I made the changes and github still thinks something has changed on this file.